tf_agents/cogment_verse_tf_agents/reinforce/training_run.py

```diff
@@ -78,7 +78,16 @@ async def training_run(run_session):
                 },
                 **model_kwargs,
             )
-            run_xp_tracker.log_params(model._params, config, batch_size=batch_size)
+
+            run_xp_tracker.log_params(
+                player_count=config.player_count,
+                batch_size=batch_size,
+                model_publication_interval=config.model_publication_interval,
+                model_archive_interval_multiplier=config.model_archive_interval_multiplier,
+                environment=config.environment_name,
+                agent_implmentation=config.agent_implementation,
+                **{k: v for k, v in model._params.items() if k != 'model_params'},
```

Comment:
thanks to one of the latest update to the xp tracker you can now simply do 
```suggestion
                {k: v for k, v in model._params.items() if k != 'model_params'},
```

but the schedule is in there too (which make that crash) and the fact that you need to skip 'model_params' feels like a smell :)